nsqadmin/http.go

```diff
@@ -1139,6 +1184,51 @@ func (s *httpServer) statisticsHandler(w http.ResponseWriter, req *http.Request,
 		messages = append(messages, pe.Error())
 	}
 
+	if filter == "channel-timeout" || filter == "channel-requeue" {
```

Comment:
as channel stats returns as map, and sort performed against topic stats list inside GetNSQStats, hence I sort reference list to channel stats map after channel stats map returned.